brew/install-start-with-checks.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+set -eu
+
+eval $(brew/vars.sh)
+
+echo
+echo brew version...
+brew --version
+
+set +e # to turn -e off (relax this only for time being)
+brew doctor # TODO(mkramlich): for brew maintainer, best if have no findings and exit 0
+set -e # to turn back on
+
+echo
+echo audit formula...
+brew audit --strict --online --formula $TS_FORMULA # keep this run output blank and exit 0
+
+echo
+echo installing...
+#brew install -v --build-from-source $TS_FORMULA
+brew install -v --formula $TS_FORMULA
+
+echo
+echo checking exe location and diffs...
+diff $TS_BIN/tailscale $TS_CELLAR/$TS_VER/bin/tailscale
+diff $TS_BIN/tailscaled $TS_CELLAR/$TS_VER/bin/tailscaled
+
+echo
+echo check if brew has formula...
+brew list tailscale
+
+echo
+echo check if installed formula matches orig...
+diff $TS_FORMULA $TS_CELLAR/$TS_VER/.$TS_FORMULA
+
+echo
+echo lint of the generated launchctl plist for homebrew tailscale...
+# should print "<filepath>: OK" and exit 0; brew services (or launchctl) below
+# might reject (or silently allow) any given !0 case so better to catch an issue
+# early/wider than not, and automate extra clues for maintainer
+plutil -lint $TS_CELLAR/$TS_VER/$TS_PLIST
+
+echo
+echo brew test before starting...
+brew test $TS_FORMULA
+
+echo
+echo starting...
+sudo brew services start tailscale
+
+echo
+echo pausing as tailscaled starts...
+sleep 8
+
+echo
+echo checking that our registered Global Daemons plist matches expected...
+diff $TS_CELLAR/$TS_VER/$TS_PLIST /Library/LaunchDaemons/$TS_PLIST
```

Comment:
Is this path writable on Big Sur?

Reply:
can't say directly. will cite my other comment about platform testing so far

however in this case, these are the paths that brew was using, and when using brew services to register a service your host's brew is responsible for putting things in the right locations for the host's OS. so this was just a quick sanity check I added. I can drop it or adjust as proves necessary